DNN Platform/Library/DotNetNuke.Library.csproj

```diff
@@ -76,8 +76,8 @@
   </PropertyGroup>
   <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.Targets" />
   <ItemGroup>
-    <Reference Include="BouncyCastle.Crypto, Version=1.8.9.0, Culture=neutral, PublicKeyToken=0e99375e54769942, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Portable.BouncyCastle.1.8.9\lib\net40\BouncyCastle.Crypto.dll</HintPath>
+    <Reference Include="BouncyCastle.Crypto, Version=1.8.8.0, Culture=neutral, PublicKeyToken=0e99375e54769942, processorArchitecture=MSIL">

```

Comment:
Any reason for lowering the version or is that a mistake ?

Reply:
The version that MailKit and MimeKit were depending on was 1.8.8. With 1.8.9 in there .NET Kept throwing errors trying to load the dll. Wasnt until I went to 1.8.8 that the loading errors stopped.

Reply:
Interesting... Ok, makes sense then...